stdlib/public/core/Span/Span.swift

```diff
@@ -200,6 +200,28 @@ extension Span {
     // lifetime of 'buffer'. Make the dependence explicit.
     self = unsafe _overrideLifetime(span, borrowing: buffer)
   }
+
+  @_semantics("fixed_storage.check_index")
+  @inline(__always)
+  @_alwaysEmitIntoClient
+  internal func _checkIndex(_ position: Index) {
+    _precondition(indices.contains(position), "Index out of bounds")
+  }
+
+  /// Accesses the element at the specified position in the `Span`.
+  ///
+  /// - Parameter position: The offset of the element to access. `position`
+  ///     must be greater or equal to zero, and less than `count`.
+  ///
+  /// - Complexity: O(1)
+  @_alwaysEmitIntoClient
+  public subscript(_ position: Index) -> Element {
+    //FIXME: change to unsafeRawAddress when ready
+    unsafeAddress {
+      _checkIndex(position)
+      return unsafe _unsafeAddressOfElement(unchecked: position)
+    }
+  }
```

Comment:
This is a subscript that requires copyable elements. I think the only needed change was to remove the `_checkIndex()` overload at line 447. Both overloads of the subscript have a reason to be there.

Reply:
Ah okay.. I removed the _checkIndex overload.